src/mscorlib/src/Microsoft/Win32/SafeHandles/SafeViewOfFileHandle.cs

```diff
@@ -22,25 +22,5 @@
 
 namespace Microsoft.Win32.SafeHandles
 {
-    internal sealed class SafeViewOfFileHandle : SafeHandleZeroOrMinusOneIsInvalid
```

Comment:
Ditto.  Anywhere all of the code is being removed, we should be able to instead just remove the whole file.